docs/extend/extend-cli/cli-developing-a-plugin.md

```diff
@@ -1,219 +1,348 @@
+    };
+
+    export = IssueDefinition;
+    ```
+
+4. In the `diff` folder, create a folder named `data-sets`.
+5. In the `data-sets` folder, create the following two files: 
+    - `DataSets.definition.ts`
+    - `DataSets.handler.ts`
+
+6. Copy and paste the following code into the `DataSets.definition.ts` file:
+    ```typescript
+    import { ICommandDefinition } from "@zowe/imperative";
+
+    export const DataSetsDefinition: ICommandDefinition = {
+        name: "data-sets",
+        aliases: ["ds"],
+        summary: "data sets to diff",
+        description: "diff the first data set with the second",
+        type: "command",
+        handler: __dirname + "/DataSets.handler",
+        positionals: [
+            {
+                name: "oldDataSet",
+                description: "The old data set",
+                type: "string"
+            },
+            {
+                name: "newDataSet",
+                description: "The new data set",
+                type: "string"
+            }
+        ],
+        profile: {
+            optional: ["zosmf"]
+        }
+    };
+    ```
+
+7. Copy and paste the following code into the `DataSets.handler.ts` file:
+    ```typescript
+    import { ConnectionPropsForSessCfg, ICommandHandler, IHandlerParameters, ISession, Session } from "@zowe/imperative";
+    import { DataSetDiff } from "../../../api/DataSetDiff";
+
+    export default class DataSetsDiffHandler implements ICommandHandler {
+        public async process(params: IHandlerParameters): Promise<void> {
+            const sessCfg: ISession = {
+                hostname: params.arguments.host,
+                port: params.arguments.port,
+                basePath: params.arguments.basePath,
+                rejectUnauthorized: params.arguments.rejectUnauthorized,
+                protocol: params.arguments.protocol || "https"
+            };
+            const sessCfgWithCreds = await ConnectionPropsForSessCfg.   addPropsOrPrompt<ISession>(sessCfg, params.arguments,
+                { doPrompting: true, parms: params });
+            const session = new Session(sessCfgWithCreds);
+
+            const resp = await DataSetDiff.diff(session, params.arguments.oldDataSet, params.arguments.newDataSet);
+            params.response.console.log(resp);
+        }
+    }    
+    ```
+    When successful, the `Diff.definition.ts`, `DataSets.definition.ts`, and `DataSets.handler.ts` files contain the new code.
+
+**Note:** If you are adding multiple commands to your CLI plug-in, consider moving the code that creates a session into a base handler class that can be shared across multiple commands. See the [sample plugin code](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/cli/list/ListBaseHandler.ts) for an example of how this can be done.
+
+**To create a unique plug-in:** Refer to file names specific to your project. Your code likely follows the same structure, but command names, descriptions, and options would differ.
```

Comment:
¿maybe instead?: **To create a unique plug-in:** Refer to file names specific to your project. Your code likely follows the same structure, but the command name, <ins>handler, and definition</ins> would differ.

Reply:
Updated. Thank you!